src/constants/index.tsx

```diff
@@ -2,34 +2,42 @@ import Fantom from '../assets/networks/fantom-network.jpg'
 
 export enum ChainId {
     MAINNET = 250,
+    TESTNET = 0xfa2,
 }
 
 export const NETWORK_ICON = {
     [ChainId.MAINNET]: Fantom,
+    [ChainId.TESTNET]: Fantom,
 }
 
 export const NETWORK_LABEL: { [chainId in ChainId]?: string } = {
     [ChainId.MAINNET]: 'Fantom',
+    [ChainId.TESTNET]: 'Fantom',
 }
 
 export const RARITY_CONTRACTS = {
     [ChainId.MAINNET]: '0xce761d788df608bd21bdd59d6f4b54b2e27f25bb',
+    [ChainId.TESTNET]: '0xce761d788df608bd21bdd59d6f4b54b2e27f25bb',
 }
 
 export const GOLD_CONTRACTS = {
     [ChainId.MAINNET]: '0x2069B76Afe6b734Fb65D1d099E7ec64ee9CC76B2',
+    [ChainId.TESTNET]: '0x2069B76Afe6b734Fb65D1d099E7ec64ee9CC76B2',
 }
 
 export const ATTRIBUTES_CONTRACT = {
     [ChainId.MAINNET]: '0xB5F5AF1087A8DA62A23b08C00C6ec9af21F397a1',
+    [ChainId.TESTNET]: '0xB5F5AF1087A8DA62A23b08C00C6ec9af21F397a1',
 }
 
 export const MULTIADVENTURE_CONTRACT = {
     [ChainId.MAINNET]: '0x0D4C98901563ca730332e841EDBCB801fe9F2551',
+    [ChainId.TESTNET]: '0x0D4C98901563ca730332e841EDBCB801fe9F2551',
 }
 
 export const DAILYCARE_CONTRACT = {
     [ChainId.MAINNET]: '0xf1bf34E46ECf465591B7a7fA9635E4C583174fa3',
+    [ChainId.TESTNET]: '0xf1bf34E46ECf465591B7a7fA9635E4C583174fa3',
 }
 
```

Comment:
I think rarity is not deployed on fantom testnet

Reply:
Since Rarity source code is available on Github, I'll see if I can deploy it on testnet so that we don't have to spend unnecessary FTM when testing it during development

Reply:
Okay I understand. I've already removed it